### PR TITLE
Feature/microservices

### PR DESCRIPTION
**Pull request by SaintUkrainian**

### CODE REVIEW

File: person-service/src/main/java/com/github/saintukrainian/cloudcrud/service/PersonService.java [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+  /**
+   * Method for getting a person with details by id
+   *
+   * @param id person id
+   * @return person with details
+   */
+  public PersonWithDetails getPersonWithDetailsById(int id) {
+    SpannerQueryOptions queryOptions = new SpannerQueryOptions();
+    Statement statement = Statement.newBuilder(PERSON_WITH_DETAILS_SQL).bind("id").to(id).build();
```
Comment (vlish) [inline person-service/src/main/java/com/github/saintukrainian/cloudcrud/service/PersonService.java:231]: Every chain method call from new line.

File: person-service/src/main/java/com/github/saintukrainian/cloudcrud/spannerconfig/SpannerRemoteConfig.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties("spanner")
+public class SpannerRemoteConfig {
+  @Getter @Setter private String instanceId;
```
Comment (vlish) [inline person-service/src/main/java/com/github/saintukrainian/cloudcrud/spannerconfig/SpannerRemoteConfig.java:11]: Is it possible to not duplicate getter and setter annotations for every field.

File: person-service/src/test/java/com/github/saintukrainian/cloudcrud/controllers/AbstractTest.java [hunk 3]
```diff
@@ -21,14 +20,13 @@
 public abstract class AbstractTest {
 
   protected MockMvc mvc;
-  protected static final String PERSONS_URL = "http://localhost:8080/persons/";
-  protected static final String PWD_URL = "http://localhost:8080/pwd/";
-  protected static final String PD_URL = "http://localhost:8080/pd/";
+  protected static final String PERSONS_URL = "http://person-service/persons/";
+  protected static final String PWD_URL = "http://person-service/pwd/";
```
Comment (vlish) [inline person-service/src/test/java/com/github/saintukrainian/cloudcrud/controllers/AbstractTest.java:24]: Just as suggestion for future - use more informative names, for example persons-with-details.

File: person-service/src/main/java/com/github/saintukrainian/cloudcrud/service/PersonService.java (hunk 1 above)
Comment (SaintUkrainian) [reply person-service/src/main/java/com/github/saintukrainian/cloudcrud/service/PersonService.java:231]: Google formats it to be like that.

File: .gitignore [hunk 4]
```diff
@@ -1,38 +1,5 @@
-HELP.md
```
Comment (olegkoskin) [inline .gitignore:1]: Why do you remove almost everything from `.gitignore` file? Do you really need files like `.iml`?

Comment (SaintUkrainian) [reply .gitignore:1]: I guess, i've removed those lines by accident.

**Pull request merged**